bouncer/shared/create_and_delete_multiple_orders.ts

```diff
@@ -1,54 +1,86 @@
-import { Keyring } from '../polkadot/keyring';
-import { handleSubstrateError, lpMutex } from '../shared/utils';
+import assert from 'assert';
+import { createLpKeypair, handleSubstrateError, lpMutex } from '../shared/utils';
 import { getChainflipApi, observeEvent } from './utils/substrate';
 import { limitOrder } from './limit_order';
+import { rangeOrder } from './range_order';
+import { depositLiquidity } from './deposit_liquidity';
+import { deposits } from './setup_swaps';
 
-export async function createAndDeleteMultipleOrders(numberOfOrders: number) {
+const DEFAULT_LP: string = '//LP_3';
+
+async function countOpenOrders(baseAsset: string, quoteAsset: string, lp: string) {
+  await using chainflip = await getChainflipApi();
+  const orders = await chainflip.rpc('cf_pool_orders', baseAsset, quoteAsset, lp);
+  if (!orders) {
+    throw Error('Rpc cf_pool_orders returned undefined');
+  }
+  let openOrders = 0;
+
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.limit_orders.asks.length || 0;
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.limit_orders.bids.length || 0;
+  // @ts-expect-error range_orders does not exist on type AnyJson
+  openOrders += orders?.range_orders.length || 0;
+
+  return openOrders;
+}
+
+export async function createAndDeleteMultipleOrders(numberOfLimitOrders: number) {
   console.log(`=== cancel_orders_batch test ===`);
   await using chainflip = await getChainflipApi();
 
-  const keyring = new Keyring({ type: 'sr25519' });
-  keyring.setSS58Format(2112);
-  const lpUri = process.env.LP_URI || '//LP_1';
-  const lp = keyring.createFromUri(lpUri);
+  const lpUri = process.env.LP_URI || DEFAULT_LP;
+  const lp = createLpKeypair(lpUri);
+
+  await Promise.all([
+    // provide liquidity to LP_3
```

Comment:
LP_3 will be funded each time we run the test, this is slow and once we are using the new chainspec, it will be unnecessary. Shouldn't we just put this in the `setup_lp_account` function? Makes sense that we would always want to fund an LP account when making one.

Reply:
Feel free to merge it if you're ok with it Jaime

Reply:
oh i see. I thought the account was pre-funded in the chainspec. Yeah thats fine, we can move it later.